Pod/Classes/UIViewController+MenuItemImage.swift

```diff
@@ -19,4 +21,29 @@ public extension UIViewController {
             objc_setAssociatedObject(self, &MenuItemImageKey, newValue, .OBJC_ASSOCIATION_RETAIN)
         }
     }
+    
+    var menuItemDesc: String? {
+        get {
+            guard let desc = objc_getAssociatedObject(self, &MenuItemDescKey) as? String else { return nil }
+            return desc
+        }
+        set {
+            objc_setAssociatedObject(self, &MenuItemDescKey, newValue, .OBJC_ASSOCIATION_RETAIN)
+        }
+    }
+    
+    var menuItemView: MenuItemView? {
```

Comment:
This way which sets custom property is not a good one. Therefore, it shouldn't be used unnecessary property like `menuItemView`. It can be accessed by `PagingMenuController` instance.
Please revert this code, plus `updateMenuContent` method.